tests/integrations/test_nongated_experts.py

```
@@ -0,0 +1,162 @@
+        for k in range(K):
+            e = int(idx[t, k])
+            h = x[t] @ w1[e].T
+            a = F.relu(h).square()
+            out[t] = out[t] + wts[t, k] * (a @ w2[e].T)
+    return out
+
+
+class TestNonGatedGroupedMLP:
+    def test_forward_matches_oracle(self):
+        torch.manual_seed(0)
+        E, L, I, T, K = 4, 8, 12, 16, 2
```

Ruff (0.15.21)

[error] Ambiguous variable name: `I`

(E741)